Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build
+FROM rust:1.73 AS build
+
+WORKDIR /grafbase
+
+COPY ./cli ./cli
+COPY ./engine ./engine
+
+COPY ./cli/Cargo.lock ./cli/Cargo.lock
```

Comment:
Unless I'm missing something, copying the cargo files after having copied the whole engine and cli directories has no use. Any change to the cli cargo files will make docker start from the first `COPY ./cli ./cli` anyway.

Reply:
Yeah, removing for now. The idea is to cache the dependencies layer, but we can come back to that later.